.github/workflow/python-app.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+
+name: Python application
```

Comment:
We will likely need brian to enable these and in the future move to micro-nova's github so we don't use up his minutes.